### PR TITLE
Hulk Rework 2021 Edition [BOUNTY] [Old-Yogs]

### PR DESCRIPTION
**Pull request by Bibby0110**
https://github.com/yogstation13/yogstation-2017-/pull/2762

# General Documentation

### Intent of your Pull Request

- Hulks are now not pemanent
Genetics gives you power which makes you angry. If you get hurt hard enough you become enraged and transform into hulk. Overtime you calm down and become human again, but if you punch stuff or someone hurts you more you will stay as hulk longer. Hugging Hulk will calm him down faster :)
Btw that 'rage system' is based on stamina and disablers now have use against hulks!

- Hulks rip appart jumpsuits and drop outer clothes on transformation
Destroys any jumpsuit worn buy a person transforming into a hulk. Spawns a few cloth scraps when jumpsuit is destroyed. Drops outer suit on the ground. Disallows hulks to wear any jumpsuit/outer clothes when they are mutated.
(Wizards with Mutate spell wont get their clothes dropped/destroyed)
Also removes undershirt to show off those thick muscles!

- Hulks get brain damage
- DISCLAIMER: They get enough damage to get mild traumas (fuk powergamers), still can't use machinery
Now you cant use computers and complex machinery as Hulk, also you get some nice Hulk's favorite phrases gibbering during your green rage periods!

- Hulk special abilities and buffs
Now you have Hulk Smash. Hulk smashes floor under him knocking away anyone standing too close.
And hulk cant be slown down till he in crit.

- Hulk can enter Enraged state
if Hulk is not calmed down enough he has a chance to become enraged. When this happens Hulk starts to move almost uncontrollably and applies punches to everything he bumps into. Luckily this enragement periods don't last long
### Why is this change good for the game?

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
See above

### What general grouping does this PR fall under? 
Balance changes

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
No

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
Not important

# Changelog

:cl:  
rscadd: NT Genetics division reports that Hulk mutation power was altered in an accident which involved genetics assistant, beaker of unstable mutagen and a banana peel.
tweak: Hulk power is not permanent now. To become Hulk you need to get a power from genetics and then get hurt hard enough.
rscadd: Hulks can be calmed down over time or by using disablers so they will become human again. Alternatively Hulks who attack stuff or take damage will become more enraged and will stay green for longer.
rscadd: Hulk muscles now grow big so they rip apart jumpsuits and drop outer clothes and they wont be able to wear those clothes until they are human again.
rscadd: Hulks get new ability to smash the ground underneath them and knock back anyone nearby!
rscadd: If hulk is not calmed down he can spontaneously become enraged. When Hulk is enraged he moves uncontrollably and punches everything in his way.
tweak: Hulks now get brain damage and they can't be slowed down on low health.
/:cl:

### CODE REVIEW

**Comment by MegaEmpirical** [comment]
Why hello there, how you doin?

**Comment by JackKindred** [comment]
hell yeah

**Comment by Szyszkrzyneczka** [comment]
Merge this
hulk smash

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 563c7ee6411d75f33c7936058ab7dfa1df6212ee
Pull Request: 9cf7141afe5cb495635b1a74837692f531b78edb
Server: 9997d14fb83ac87471f15e84db5fcda80f54c8ae

**Comment by Tipy1802** [comment]
Does not work as intented 
There are 3 hulk mutations
The first one is made the old way by mixing strenght and radioctivity and acts like the old hulk mutation (permenant and without the changed)
The second one is called ''Hulk'' and is found like all mutations, by being found in someone's DNA and not as a combination
However it does not appear to work, even after getting hurt to almost crit it did not activate
The third one is called ''Hulk State'' found the same way as ''Hulk'' and just like it doesn't work

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 9a701eba50318f4c0312e250e2787be3f16666c7
Pull Request: 9cf7141afe5cb495635b1a74837692f531b78edb
Server: b9c5898e437f0accb0a02c5dbcce953a534e5ef7

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: d59ad420f28f008259595b77e8b12a350beb767e
Pull Request: 9cf7141afe5cb495635b1a74837692f531b78edb
Server: 02a62650a0822425ee1fb1470354c531afd2b78b

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 7581207a31b3880529365745c29f53c056473a97
Pull Request: 9cf7141afe5cb495635b1a74837692f531b78edb
Server: e7c8195d5fdd3f35b25455c7a7bae43c7c6c3900

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: cab8715cc55ffa6b9e7be14ac62f95fc0e754b56
Pull Request: 9cf7141afe5cb495635b1a74837692f531b78edb
Server: 213a014f66f5d2974420815ae872304fc23c53bd

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 61edbfb9bb993c220b540e805fbba28a81be1a09
Pull Request: 9cf7141afe5cb495635b1a74837692f531b78edb
Server: b45645c36c298178717f1a71977c45ff1650a1aa

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 9be1f32b7dc56623b6f6ca060b5e2f24765b0082
Pull Request: 9cf7141afe5cb495635b1a74837692f531b78edb
Server: cb876944d10878e0ab8cb4f9aeca0a31260d83aa

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 19930dd420bc1ac6cc69a724f0e72b1cfd2e9468
Pull Request: 9cf7141afe5cb495635b1a74837692f531b78edb
Server: 234c73fd0af454508ba75a582f4d5fad802bdb6a

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 1c7196a267856c2a0914cd975779cfbef4e41657
Pull Request: 9cf7141afe5cb495635b1a74837692f531b78edb
Server: ed1b7b319c6f39ac8a5e9eb3887060119a504840

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 5d42ae4ef1bc9279a8880084b8f70f7f66abab6c
Pull Request: 9cf7141afe5cb495635b1a74837692f531b78edb
Server: 06f47864942a03957a97b515041532fd05413f20

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 5d42ae4ef1bc9279a8880084b8f70f7f66abab6c
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 03b23a2b8e6b5a24544b9dd43a963230b9ae7b3d

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 686285aa7315617df9b57670abe3cd69b97b2b0f
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 27feab21df62b2009a118d8f0108a684ba5f22e5

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 8e22985088efb49a12ed8a7ddf77923fbc806eaf
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 2e08e4b858f1dab2d8936cc5c44b0c1528915224

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 9ed8886a0114c351d7225afbb7c3f42d6fb86282
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 78eaea031cfd1f543118ea68d7dbb28cf1f60d85

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 6dfe6e4c25a2ef499385d4d7d3d22dac2f84c8ef
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 9e5251feed6b12058278cbff4a94b8eac6e61e25

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 43ba23fd47ed7ac7b0ae2fab3298c50bf01a716f
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 4ed606f765b1e1c88754c21fa06855c6deeccf53

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: b91a4e6f0eadc5ae9fe670a84c2ea727218049d1
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: eb877593dab0a999c1c860f75fbe25571c64c45f

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: f3db06abdb8f41250cf2596d2eb423217619b576
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: ef95f2f63b0e15741da9645d0cf00e6530505b5d

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 7a02758356782f52673e63881daafe29cf72905d
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: ed1c11085eccb8337d29d28e4aa33a706bb6b653

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: ea22c2460820cc589cd52d975cc180f161d87be9
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: c4aa431f16fd120a9c812bf88624f38f5fb16438

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 6a959204098b1add1d64cddf7c78c6f5c4e9e262
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 56a86f26a74ad52c28e7ceb6fdba24a83ebe113d

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 288c9155d93609e5d4f6e672ca4dc593ad0498e9
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: af6769dfe0cb80f92a7c0286198680db6a9faa8a

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 6a1126f38178bbea9a6ea1232e3667328d8adf36
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 737cce2c7d87db0cc9daa543f7ed399f7d3aeb06

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 5894ffa2d8dd7850499ca024060d6aa1cce58820
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 266965157c9027449280cc5051fd32e58b2e78e9

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 663db5ba87e3f70b4bfadfbec0b8084993e025cc
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 13bb01a3ea9296c5a0b36d006a10864b0a560c01

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 08b8ca1c963abb737a46b793f5a63dc417b11692
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 7dfb6d8475534fd298febdbbec1fe2f8492ed93e

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 8b76fb771ad29853581045d15c3e00d74ae8cda4
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 4bb363b298075454447209e1f222048b569b1d31

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: c4b77043250a5249b0b3a6f6b9140bda978100bc
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: fb408af13ae44b02478fd88590d063f0bb8587ba

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 4caae8b90c864d146d0837c3858650b0d6fce598
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 3156df07b0e885f446540d6df824f056175c7ac5

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: dc9f14a06bbc12873c7b953fc1d8819ff123f9a0
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: b9ea523cbe5a645f0a8051e3198e2d826d762af1

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: ab1e72c970e7dbee30a5889fe87efa90a50f439d
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 556a106f02401c1f1540f605554f209856653006

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 35a1b736fd9b4cc3e797190ca7d0edda55aba8ca
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: e34f5fadd1f263a568b9cf4cc9cfb5fdded8e3f2

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 42ddd6b7f84a9f159a2c885362d1022469990b63
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 1188cf5a4fe1f8713a812131e806a23a4d3d4a89

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 7bfa69681d362a0efaacb2371dbccac81a015858
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 8b3744ba06ec12b44ddb4444bcb4dbef34455f72

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 2f17de354be5a09999c1c42be85a396934c2db94
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 0c99b9bd441b6abfb305097763742ad26dd29db0

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 9774ed6aca76387fd0486662f7ec8b74864ed0bf
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 9867d468da47ee4bfe23d4793eaec78ff1bd116b

File: code/datums/mutations/hulk.dm [hunk 1]
```diff
@@ -44,3 +45,74 @@
...
+
+/datum/mutation/human/active_hulk/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+	owner.SetParalyzed(0)
+	ADD_TRAIT(owner, TRAIT_STUNIMMUNE, TRAIT_HULK)
+	ADD_TRAIT(owner, TRAIT_PUSHIMMUNE, TRAIT_HULK)
+	ADD_TRAIT(owner, TRAIT_IGNORESLOWDOWN, TRAIT_HULK)
+	RegisterSignal(owner, COMSIG_MOB_SAY, .proc/handle_speech)
+	owner.mind.AddSpell(new /obj/effect/proc_holder/spell/aoe_turf/repulse/hulk(null))
```
Comment (redmoogle) [inline code/datums/mutations/hulk.dm]: make sure the mind exists or well just ? it
```suggestion
	owner?.mind?.AddSpell(new /obj/effect/proc_holder/spell/aoe_turf/repulse/hulk(null))
```

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: c10c21773b1fb4657cf18ad51f68ed114624a12f
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 86000cd97b509d82f286a17aac57e7bb6a0aea21

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: fc48dc1c031f07573883e87dafbfa85b8c00c130
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: c962d51026425f37e0522588120fc5429dfe5b6a

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 184b703bf8e5a0fbd672057d871a5f2087822f0b
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 2506601200403247596b0aa69e79e1199a1dc6ac

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 3e13a5cf49b75fbe2f60e2261cbd728328f3e04e
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: d8c06a269bc300a8e5bf03a2e4792fe2b2a27c87

**Comment by redmoogle** [comment]
Holy shit testmerge bot is annoying

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 40aa789906643fa4bee61c75515d5dda3efef02b
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 104ae9499122caf2fb54eb7304c0c14f49a3bea9

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: c1aa722c97c2a7424d7bacc538188dd9b2c3a492
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 4d03121910558fd36432e5bcdb9d5d76e68a2a9f

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: c042e8dfbac4f7e5a446b716593a82872d7bd282
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 1b12204587cf4768d80a0597b9d860353c4f00f9

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 4a8e9c37715ff1ef83bd2f3647680aed502d6504
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 5b753f01d8c6a270589bd174af4cbe3171c73c7e

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 76f1b1010b8d297118498b64bd497b3acdafcaf5
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 9ec021d72dfc22371df2510bdf95c7e151d0c14f

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 14cbb51b24bb0171dc7ff69fc9469c7dce9348a8
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: e31522ee573bff8b94843effa291e97283a5758b

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: c85b72e912d0059ad53c7a7c327d8243c2ca8ac3
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 307afbe64dcdd45e60959c0710963a0ff35ed0c7

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 70f670d75eed63d52ac1434977211247cc0d1a39
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: ab2ac33ae854b589aef40487bd0fcb821063d298

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 7cd77b5cc180c5dd0bb7245e7f1751a967e6b98d
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 9c1fbe38ec650e63962cf896982646c5dbb3b1b8

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: f87ad2b83c5aae4b6e13dbc19315708a4d83e493
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: f179960dc865800675dfc54aa2689533c4c9a5a3

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 134f8546fa6a282a1aadb2445c5d108cd4a7da9d
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 821fee12c00215f7e4c1bac1767a5eb5c6acaa03

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: c009474a30a84fa5b4e8db311a2c06f3181b9427
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 6adf7deb7b0aeabb86f40e67e6b93060b27c97d1

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 5fda23037c8d1b311244cd8a185bbbe9f5ed275e
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: bc4b7c3311f8a0a237e503e2b8958878026007b0

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: f0a7b2afffb788a82e01ae1256ff8fc158728f22
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: a451991644b83e18118a4540527cce1c1d0024b4

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: c0762a52035e1f6c4fc2172006a3c7f8b928f851
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: dd9e749d7029a82e9d202e61b545b0a631939fc2

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: efe539f4187ed59a2c5ee4c25bd0197913af1bc9
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 61a7a5ec7a409543c9b8fad227481d7870dd5cb2

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 2f4748120c4f828422a65ff471655c49e0ae77de
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: e1c864b7ccc5bb9673a8095c016400c4bad92f37

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: fff1770e8b544be509edef7c14900647e0afebc6
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: d81efdab5a13fe2da298188860595f54b400c6dc

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: fff1770e8b544be509edef7c14900647e0afebc6
Pull Request: 10798a62545ff93ddd51e52ac66b8d3b336e89da
Server: 47528dc0e18249af8fea62ba7c0d7339a00763f3

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: fff1770e8b544be509edef7c14900647e0afebc6
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: 02583086b1247c9a1be51165afc516aad0e5e7a7

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 17d3b574ef653d3278c26a59dc8418bee32d4849
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: 3dc4353d50b708e98f7a70c983f21bd3009239e1

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: b282edb69de8b2d8900b7573ebac641561d392d3
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: 461b198c2ad376e0a8dcba584d5cabc8cd93b91d

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: f1ccd63a36b575f6b21d51678f597a016dea4341
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: 9ae5169f85f4615a97cfce794663b0fcd3630291

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: f1ccd63a36b575f6b21d51678f597a016dea4341
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: ff51cbb8edef33f36a4b21e1679264ce72f404e8

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 77b18567de776b91f70821b0cb89efcea50e5638
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: 74d901727666ecdb4a83807850b62b7ae4596cc2

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: d9fa53dbf20373162cd911ee6c408579cbb08dd4
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: 71059923354c07cd59b391346f037e2cc0a88638

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: d9fa53dbf20373162cd911ee6c408579cbb08dd4
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: e241a029bff0a862598d4cf5330270ab7a61901c

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: ef0db3a25ba81dee1c39ea8fc8162a7a4a752ce0
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: aaf2a4de3a62a53112d314c3f9281454f06ce656

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 61be296c7775d9ea88e9b8a729456b211e4cc029
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: 7556b2e6eebbdd8cd977546ea7280d0bb63bd23f

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 61be296c7775d9ea88e9b8a729456b211e4cc029
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: 30cee7883419fedc2b14d05329f2362084d4a522

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 846b6d0591d4fdfa09eb9564a32ffc4691283565
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: a3380909465cd8b2082b69723d93038188c356e5

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 846b6d0591d4fdfa09eb9564a32ffc4691283565
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: 952eb07ef4c61276755bc1d018d27c948afb0e36

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 846b6d0591d4fdfa09eb9564a32ffc4691283565
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: 9a2ec9b6425c6d90690405f9c3b94f4666402c94

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: dd1d7fc33764ab1f7a35cf63a176bb8ea99c5bb7
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: 14cdaf78532bb345dde5558d0304548c1bc44f79

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: dd1d7fc33764ab1f7a35cf63a176bb8ea99c5bb7
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: 38094ac42f2645436ff266e4acf28dbe0a1101bc

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: a6e2415c469ba1af6e29b54aa239ed65b8c36ee7
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: d24b215e0e8c6c5423888ae49e4f29a9cf171b3c

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 66b7150aeab900c3f31c14b1bd1e42b0bf923c18
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: ef2e93fba57b1a0477b759c6d1a9a04f2aa0c565

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 158c00ddf79f3de3a5da5964b55ac17a89978863
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: b8c32c664435221e3bc7144827e3654e90e4e6dc

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: ca59602e0478d0927aad7a9b1fd4711bde1f9c82
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: c2377bfb63991a30acd46b8f81123ed6b04133b5

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 39c610f599c05ab55c9551c8328eb9cd5b838117
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: 576b980d7860d3a2d8af75956984af14a64580ef

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 39c610f599c05ab55c9551c8328eb9cd5b838117
Pull Request: 80ad369341aaecfee690248634e5f22907de09dd
Server: ce13dee3d6e20b89dc7daedfb859ee58550b7dcf

**Comment by MegaEmpirical** [comment]
Last night I hugged two hulls into submission, it was nice.

How long does it take for people to lose hulk if they're not 'maintaining' it? Felt like they were able to hold it for a long ass time.

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 2]
```diff
@@ -291,6 +291,12 @@
 	AdjustSleeping(-100)
 	AdjustParalyzed(-60)
 	AdjustImmobilized(-60)
+	if(dna && dna.check_mutation(ACTIVE_HULK))
+		if(prob(60))
```
Comment (Bibby0110) [inline code/modules/mob/living/carbon/carbon_defense.dm]:
```suggestion
		if(prob(30))
```

File: code/datums/mutations/hulk.dm [hunk 3]
```diff
@@ -44,3 +45,72 @@
...
+	owner.say("PUNY HUMANS!!")
+	owner.physiology.stamina_mod = 0.3
+	owner.update_body()
+
+/datum/mutation/human/active_hulk/on_attack_hand(mob/living/carbon/human/owner, atom/target)
+	if(prob(3))
+		owner.Jitter(10)
+	owner.adjustStaminaLoss(-0.5)
+	return target.attack_hulk(owner)
+
```
Comment (Bibby0110) [inline code/datums/mutations/hulk.dm:99]:
```suggestion

/datum/mutation/human/active_hulk/on_life()
	owner.adjustStaminaLoss(0.5)

```

**Comment by Bibby0110** [comment]
> Last night I hugged two hulls into submission, it was nice.
> 
> How long does it take for people to lose hulk if they're not 'maintaining' it? Felt like they were able to hold it for a long ass time.

Nerfed hugability a bit.

Takes 100 ticks provided no smashing

File: code/datums/mutations/hulk.dm [hunk 4]
```diff
@@ -44,3 +45,75 @@
...
+/datum/mutation/human/active_hulk/on_life()
+	owner.adjustStaminaLoss(0.5)
```
Comment (Bibby0110) [inline code/datums/mutations/hulk.dm]:
```suggestion
	owner.adjustStaminaLoss(0.9)
```

**Review by Partheo — APPROVED** [review]

**Comment by AlpacaGoesCrazy** [comment]
Wow. it's nice to see my hulk rework be merged again after this much time :)

**Comment by MegaEmpirical** [comment]
> Wow. it's nice to see my hulk rework be merged again after this much time :)

I've wanted it back for a long time, glad to see it making a come back. Though sadly they had do the whole code over due to how much genetics had changed.
